packages/services/f-analytics/src/mixins/_tests/analytics.created.test.js

```diff
@@ -0,0 +1,119 @@
+import Vuex from 'vuex';
+import { shallowMount, createLocalVue } from '@vue/test-utils';
+import VueRouter from 'vue-router';
+import analyticsMixin from '../analytics.mixin.vue';
+import {
+    defaultState,
+    createStore,
+    $cookies
+} from './helpers/setup';
+
+const localVue = createLocalVue();
+localVue.use(Vuex);
+localVue.use(VueRouter);
+
+describe('Analytics', () => {
+    afterEach(() => {
+        jest.clearAllMocks();
+    });
+
+    describe('created ::', () => {
+        let component;
+        let prepareServersideAnalyticsSpy;
+        let registerStoreModuleSpy;
+        let storeUpdatePlatformDataSpy;
+        beforeEach(() => {
+            // Arrange
+            component = {
+                render () {},
+                mixins: [analyticsMixin],
+                store: createStore()
+            };
+            component.mixins[0].mounted = jest.fn(() => true);
+            storeUpdatePlatformDataSpy = jest.spyOn(component.mixins[0].methods, 'updatePlatformData').mockImplementationOnce(() => true);
+            registerStoreModuleSpy = jest.spyOn(component.mixins[0].methods, 'registerStoreModule');
+            prepareServersideAnalyticsSpy = jest.spyOn(component.mixins[0].methods, 'prepareServersideAnalytics');
+        });
+
+        it('should not attempt set the \'serverside only\' \'platformData\' properties if clientside', () => {
+            // Mocks
+            component.mixins[0].computed.isServerSide = () => false;
+
+            // Act
+            shallowMount(
+                component,
+                {
+                    localVue,
+                    mocks: {
+                        $cookies
+                    }
+                }
+            );
+
+            // Assert
+            expect(registerStoreModuleSpy).toHaveBeenCalled();
+            expect(prepareServersideAnalyticsSpy).toHaveBeenCalled();
+            expect(storeUpdatePlatformDataSpy).not.toHaveBeenCalled();
+        });
+
+        it('should set the \'serverside only\' \'platformData\' properties with appropriate defaults if serverside but values not available', () => {
```

Comment:
Doesn't have to change but you could use back ticks here instead of the escapes.

Reply:
I had that before but on the test output if the value between the back ticks is not a value that can be resolved it displays a warning indicating that it cannot find the value, e.g. Cannot find `serverside only`.  So did it this way.